CONTRIBUTING.md

```diff
@@ -881,8 +881,8 @@ Upon instantiation the base exchange class takes each URL from its list of endpo
 ### Docstrings
 
 - when a method takes another parameter as a property on params (ex. `params['something']`) add that parameter to the docstring, as params.something
-   - if that parameter is required, the type is `{str}`, `{int}`, `{etc}`, if it's optional the type is `{str|undefined}`, `{int|undefined}`, `{etc|undefined}`
-- when a parameter's default value is `undefined`, but the method contains something like `if (symbol === undefined) { throw new ArgumentsRequired('...')}`, then set the type of that parameter as `{str}`, `{int}`, `{etc}`. If an error is not thrown, then the type is `{str|undefined}`, `{int|undefined}`, `{etc|undefined}`
+   - if that parameter is required, the type is `{string}`, `{int}`, `{object}`, `{etc}`, if it's optional the type is `{string|undefined}`, `{int|undefined}`, `{etc|undefined}`
```

Comment:
@pcriadoperez, this is tricky we still have those types (int, string, dict) for Python. We should probably refactor this section a bit to handle all languages Js, Python and PHP

Reply:
As this is just for contributing and it's done all in JS, I would think it's ok.
Or maybe I should add a section specifying to what each type transpiles to for each language?